aeron-client/src/main/c/aeron_cnc.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2014-2020 Real Logic Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef AERON_AERON_CNC_H
+#define AERON_AERON_CNC_H
+
+#endif //AERON_AERON_CNC_H
```

Comment:
Why have an empty header?

Reply:
It has some definitions, which I later removed.  Will delete.